classes/submission/SubmissionDAO.inc.php

```diff
@@ -813,31 +1032,52 @@ function getActiveSubmissions($contextId = null, $submissionId = null, $title =
 		if ($sectionId) $params[] = (int) $sectionId;
 		if ($editor) array_push($params, (int) ROLE_ID_MANAGER, (int) ROLE_ID_SUB_EDITOR, $editorQuery = '%' . $editor . '%', $editorQuery);
 
+		$searchWhere = '';
```

Comment:
This is a common enough pattern that it might warrant a helper function rather than copypasta.